src/Microsoft.AspNet.Cors.Core/project.json

```diff
@@ -1,7 +1,12 @@
 {
-	"version": "1.0.0-*",
-	"dependencies": {
-	},
+    "version": "1.0.0-*",
+    "dependencies": {
+         "Microsoft.Framework.ConfigurationModel": "1.0.0-*",
+         "Microsoft.Framework.DependencyInjection": "1.0.0-*",
```

Comment:
You should only depend on Microsoft.Framework.DependencyInjection.Interfaces and Microsoft.Framework.ConfigurationModel.Interfaces